Freetype no longer ships freetype-config --  use pkg-config directly.

--- configure.orig	2021-08-06 10:05:39 UTC
+++ configure
@@ -15925,9 +15925,9 @@ fi
     LIBS="$OLDLIBS"
 fi
 	if test "$enable_clanDisplay" != "no"; then
-    OLDLIBS="$LIBS"; LIBS="`freetype-config --libs` "; { $as_echo "$as_me:${as_lineno-$LINENO}: checking for ttf" >&5
+    OLDLIBS="$LIBS"; LIBS="`pkg-config freetype2 --libs` "; { $as_echo "$as_me:${as_lineno-$LINENO}: checking for ttf" >&5
 $as_echo_n "checking for ttf... " >&6; }
-    OLD_CXXFLAGS="$CXXFLAGS"; CXXFLAGS=" `freetype-config --cflags`";
+    OLD_CXXFLAGS="$CXXFLAGS"; CXXFLAGS=" `pkg-config freetype2 --cflags`";
     if test "$cross_compiling" = yes; then :
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
